frontend/src/modules/positions/components/NewPosition.jsx

```diff
@@ -95,7 +96,27 @@ class NewPosition extends React.Component {
             return errorMessage ? `${cur.label}: ${errorMessage}` : acc
         }, false)
     setInstructor = name => this.setState({ instructor: name })
-    
+      
+    handleForce = data => {
+        for(var i = 1; i < data.length; i++) {
+            this.setState({
+                course_code: data[i][0],
+                course_name: data[i][1],
+                cap_enrolment: data[i][2],
+                duties: data[i][3],
+                qualifications: data[i][4],
+                instructor: data[i][5],
+                session_id: data[i][6],
+                hours: data[i][7],
+                openings: data[i][8],
+                start_date: data[i][9],
+                end_date: data[i][10]
+            });
+            this.props.importNewPosition(this.state)
+        }
+        
+    }
+
```

Comment:
I'm a bit lost here, why do we need to save the position information in the state before we dispatch it as an action? Could we just directly dispatch the position?  It seems like the position information will be lost in the next iteration

Reply:
Yeah, I could directly dispatch the position. I will change it.